azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/BatchAccount.java

```diff
@@ -66,13 +64,11 @@
     /**
      * @return the access keys for this batch account
      */
-    @LangMethodDefinition(AsType = LangMethodType.Method)
     BatchAccountKeys keys();
 
     /**
      * @return the access keys for this batch account
      */
-    @LangMethodDefinition(AsType = LangMethodType.Method)
     BatchAccountKeys refreshKeys();
```

Comment:
actually, the annotations that force the converter to convert this to a method, not property, should stay in place - just be replaced with @Method (recent enhancement). Any method that is parameter-less and does not start with "get", but is still truly a method (i.e. calls Azure) rather than a property would need that kind of annotation. In BatchAccount, i see the following that I suspect fal into that explicit @Method bucket:
- refreshKeys()
- synchronizeAutoStorageKeys()
- refreshKeys()

the rest appear to be ok as properties.